subsys/net/ip/net_if.c

```diff
@@ -261,6 +261,10 @@ static bool net_if_tx(struct net_if *iface, struct net_pkt *pkt)
 		net_if_tx_lock(iface);
 		status = net_if_l2(iface)->send(iface, pkt);
 		net_if_tx_unlock(iface);
+		if (status < 0) {
+			NET_WARN("iface %d pkt %p send failure status %d",
```

Comment:
This commit causes stack overflow on my esp32 app, I find it interesting that it git bisect found this commit as it seems harmless, I believe I have a rather big stack size, increasing didn't change the issue, should I report in a separate issue?

Reply:
In case someone else faces this, increasing default to CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1500 fixes the issue